src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/ModelBinder.kt

```diff
@@ -33,15 +34,12 @@ class ModelBinder(private val serializer: Serializer = MontaguSerializer.instanc
         @Suppress("UNCHECKED_CAST")
         val klass = model::class as KClass<Any>
         val errors = mutableListOf<ErrorInfo>()
-        val name = property.name
+        val name = serializer.convertFieldName(property.name)
```

Comment:
Up until now all the fields have been single words, like `username`, so we hadn't been needing to convert them to snake_case.